Kinetic_shape_reconstruction/examples/Kinetic_shape_reconstruction/data/edge-case-test/test-2-polygons.off

```diff
@@ -0,0 +1,12 @@
+OFF
```

Comment:
The test data should not be in the examples directory.  
Real interesting example data should go in the Data repository itself.